source/views/covid/updates-container.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import * as React from 'react'
+import delay from 'delay'
+import LoadingView from '../components/loading'
+import {NoticeView} from '../components/notice'
+import {reportNetworkProblem} from '../../lib/report-network-problem'
+import {UpdatesList} from './updates-list'
+import {ButtonCell} from '../components/cells/button'
+
+import type {TopLevelViewPropsType} from '../types'
+import type {UpdateType} from './types'
+
+type Props = TopLevelViewPropsType & {
+	title: string,
+}
+
+type State = {
+	updates: UpdateType[],
+	loading: boolean,
+	refreshing: boolean,
+	error: ?Error,
+}
+export class UpdatesContainer extends React.PureComponent<Props, State> {
+	state = {
+		updates: [],
+		loading: true,
+		error: null,
+		refreshing: false,
+	}
+
+	componentDidMount() {
+		this.fetchData().then(() => {
+			this.setState(() => ({loading: false}))
+		})
+	}
+
+	fetchData = async () => {
+		try {
+			let url = 'https://www.carleton.edu/covid/wp-json/wp/v2/posts'
+			let updates: UpdateType[] = await fetchJson(url)
+			this.setState(() => ({updates}))
```

Comment:
Should datafetching be proxied by CCC Server or do we want to leave this on-device?

Reply:
Ah, the age-old debate. 

Let's proxy it, I think?

Reply:
Done!